### PR TITLE
Fixed registration typo

### LINKED ISSUE #27: [Website] Register has a typo
Registration misspelled. I also changed it to "register" so it's an active verb.

### PR DESCRIPTION
**Pull request by evankuo1**
fixed #27

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
+bin/
+obj/
+.vscode/
+.vs/
+app.db
+
+=======
```
Comment (cptchuckles) [inline .gitignore:8]:
```suggestion
```
There seems to be the remnants of a merge conflict in the `.gitignore` file, remove these artifacts

File: .gitignore [hunk 2]
```diff
@@ -455,3 +463,4 @@ $RECYCLE.BIN/
 #!.vscode/tasks.json
 #!.vscode/launch.json
 #!.vscode/extensions.json
+>>>>>>> upstream/master
```
Comment (cptchuckles) [inline .gitignore:466]:
```suggestion
```

**Review by cptchuckles — CHANGES_REQUESTED** [review]
Just gotta take some junk out of `.gitignore`

**Comment by evankuo1** [comment]
I think I fixed the .gitignore. Do I delete this pull request and submit a new one?

**Comment by cptchuckles** [comment]
No need, the PR follows your branch's commits

File: .gitignore (hunk 2 above)
Comment (cptchuckles) [reply .gitignore:458]: Still this line tho

**Comment by cptchuckles** [comment]
`.gitignore` line 458 still needs to be deyeeted

**Comment by evankuo1** [comment]
Sorry for the issue. My merge editor in VS didn't seem to want to open, so I edited it out manually and missed that line.

Hopefully it's fixed correctly this time. Sorry again for the trouble.

**Pull request merged**